src/client.cpp

```diff
@@ -31,6 +31,7 @@ CClient::CClient ( const quint16  iPortNumber,
                    const QString& strMIDISetup,
                    const bool     bNoAutoJackConnect,
                    const QString& strNClientName,
+                   const bool     bNEnableIPv6,
```

Comment:
It doesn't need the ugly `N`, generally.

Reply:
Was just following what appeared to be the majority convention. No strong view either way.

Reply:
I'm not going to block it for that, either.  (It's just on my list of things to get rid of.)